src/main/java/io/fabric8/elasticsearch/plugin/acl/UserRolesMappingSyncStrategy.java

```diff
@@ -39,15 +39,15 @@ public UserRolesMappingSyncStrategy(SearchGuardRolesMapping mapping, long expire
 
     @Override
     protected void syncFromImpl(OpenshiftRequestContext context, RolesMappingBuilder builder) {
-        
+
         final String user = context.getUser();
+        String kibanaRoleName = BaseRolesSyncStrategy.formatKibanaRoleName(context);
+        builder.addUser(kibanaRoleName, user)
+            .expire(getExpires());
         if (context.isOperationsUser()) {
-            builder.addUser(SearchGuardRolesMapping.ADMIN_ROLE, user);
-            builder.addUser(SearchGuardRolesMapping.KIBANA_SHARED_ROLE, user);
-            builder.expire(getExpires());
+            builder.addUser(SearchGuardRolesMapping.ADMIN_ROLE, user)
+                .expire(getExpires());
         } else {
```

Comment:
is it correct to have an else here? wouldn't' we want an admin to pick up the role in addition to the admin_role? we do something similar in `ProjectRolesMappingSyncStrategy.java`

Reply:
The code will setup the kibana role as well as access to the projects they can see.  In this case, admin users will always be able to see the entire set of indices.